.env.gps

```diff
@@ -9,6 +9,7 @@
 
 TRACING_STRATEGY=gps
 AUTHORITIES_YAML_ROUTE=https://raw.githubusercontent.com/Path-Check/trusted-authorities/master/staging/authorities.1.0.1.yaml
+AUTHORITIES_LIST_URL=https://raw.githubusercontent.com/Path-Check/safeplaces-frontend/develop/healthcare-authorities.yaml
```

Comment:
This should be removed from the app altogether, this is an outdated list (prior to MVP1)

Reply:
Sweet - will take care of that.